### PR TITLE
SOLR-17439 Fix missing documentation for finding out non-standard PID for Solr in Docker

### PR DESCRIPTION
**Pull request by rmueller83**
https://issues.apache.org/jira/browse/SOLR-17439

# Description

The documentation misses a section for finding out a non-default PID.

# Solution

Added a `ps` command for finding the PID.

# Tests

I successfully executed the command in a running Solr 9.6.1 and 9.7.0 container.

# Checklist

Please review the following and check all that apply:

- [x] I have reviewed the guidelines for [How to Contribute](https://github.com/apache/solr/blob/main/CONTRIBUTING.md) and my code conforms to the standards described there to the best of my ability.
- [x] I have created a Jira issue and added the issue ID to my pull request title.
- [x] I have given Solr maintainers [access](https://help.github.com/en/articles/allowing-changes-to-a-pull-request-branch-created-from-a-fork) to contribute to my PR branch. (optional but recommended, not available for branches on forks living under an organisation)
- [ ] I have developed this patch against the `main` branch.
- [ ] I have run `./gradlew check`.
- [ ] I have added tests for my changes.
- [x] I have added documentation for the [Reference Guide](https://github.com/apache/solr/tree/main/solr/solr-ref-guide)

### CODE REVIEW

File: solr/solr-ref-guide/modules/deployment-guide/pages/solr-in-docker.adoc [hunk 1]
```diff
@@ -342,6 +342,13 @@ Usage: jattach <pid> <cmd> [args ...]
     jcmd : execute jcmd command
 ....
 
+To find out the PID of Solr you can execute the following command and pick the `java` entry from the results:
```
Comment (cpoerschke) [inline solr/solr-ref-guide/modules/deployment-guide/pages/solr-in-docker.adoc:345]: Can it be assumed that `grep` is available? If it was available (I don't know if it is or isn't) then could also use `ps -ef | grep java` to reduce the list.

Comment (epugh) [reply solr/solr-ref-guide/modules/deployment-guide/pages/solr-in-docker.adoc:345]: you know, I almost suggested that!

Comment (rmueller83) [reply solr/solr-ref-guide/modules/deployment-guide/pages/solr-in-docker.adoc]: Sure, using `grep` has only one result. I updated the PR.

**Comment by janhoy** [comment]
Hi, thanks for this. It's a tiny doc change, so normally we don't attribute it in CHANGES.txt and in release notes. However, we encourage you to keep contributing and for a more substantial change you'll for sure be mentioned :) 

I'll back port this to 9x and 9.7 since our refguide docs can be fixed retroactively outside releases :)

**Pull request merged**